test/onnx/test_utility_funs.py

```diff
@@ -101,25 +99,6 @@ def forward(self, x, y, t):
         for node in graph.nodes():
             assert node.kind() != "onnx::SplitToSequence"
 
-    def test_output_list(self):
```

Comment:
Why is this test removed?

Reply:
As this test is used for handling incorrect format for `example_outputs`. If we want to remove this parameter, it might not need this any more.